lib/shelley/cardano-wallet.cabal

```diff
@@ -43,6 +43,7 @@ library
     , cardano-crypto
     , cardano-crypto-class
     , cardano-crypto-wrapper
+    , cardano-wallet-launcher
```

Comment:
That dependency seems to be in the wrong direction :thinking: ?

Reply:
It's just that the package name is misleading. We can fix that later - it's slightly easier to do without cardano-wallet-jormungandr.